deny.toml

```diff
@@ -50,14 +50,9 @@ allow = [
     "CC0-1.0",
     "ISC",
     "OpenSSL",
-    "zlib-acknowledgement",
     "Zlib",
 ]
-exceptions = [
```

Comment:
why are these removed?

Reply:
Because these exceptions were necessary only because of Tantivy. But I have temporarily removed this crate while waiting to reintroduce Tantivy with the current version of the semconvs.